dwav/cime_config/buildnml

```diff
@@ -61,7 +61,9 @@ def _create_namelists(case, confdir, inst_string, infile, nmlgen, data_list_path
     config['dwav_mode'] = dwav_mode
 
     # do not allow single column mode for dwav
-    expect(not case.get_value('PTS_MODE'),
+    scol_lon = float(case.get_value('PTS_LON'))
```

Comment:
check for None first as above.

Reply:
Good point! I've added these new consistency checks in all the data model buildnml routines.